packages/common/src/content/HubContent.ts

```diff
@@ -109,12 +110,21 @@ export class HubContent
    * @param editorContext
    * @returns
    */
-  toEditor(editorContext: IEntityEditorContext = {}): IHubContentEditor {
-    // Cast the entity to it's editor
-    const editor = cloneObject(this.entity) as IHubContentEditor;
-
-    // Add other transforms here...
-    // NOTE: Be sure to add tests for any transforms in the test suite
+  async toEditor(
+    editorContext: IEntityEditorContext = {},
+    include: string[] = []
+  ): Promise<IHubContentEditor> {
+    // 1. optionally enrich entity and cast to editor
+    const editor = include.length
```

Comment:
I'd like for @sonofflynn89 to take a look at this (after you merge) when he gets back and consider whether or not it would make sense to use this as the point to fetch the server enrichment for feature service content.